projects/packages/transport-helper/src/class-helper-script-manager.php

```diff
@@ -49,6 +49,11 @@ public static function install_helper_script( $script_body ) {
 		// Replace '[wp_path]' in the Helper Script with the WordPress installation location. Allows the Helper Script to find WordPress.
 		$script_body = str_replace( '[wp_path]', addslashes( ABSPATH ), $script_body );
 
+		$wp_filesystem = self::get_wp_filesystem();
+		if ( ! $wp_filesystem ) {
+			return new \WP_Error( 'install_faied', 'Failed to install Helper Script' );
```

Comment:
```suggestion
			return new \WP_Error( 'install_failed', 'Failed to install Helper Script' );
```

Reply:
Stupid keyboard. Or stupid fingers, not sure which. 😀